Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -19,17 +21,15 @@ namespace HoloToolkit.Unity.InputModule
     /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
     /// or in your Visual Studio Package.appxmanifest capabilities.
     /// </summary>
-    public partial class KeywordManager : MonoBehaviour
+    public partial class KeywordManager : BaseInputSource
     {
         [System.Serializable]
-        public struct KeywordAndResponse
+        public struct KeywordAndKeyCode
         {
             [Tooltip("The keyword to recognize.")]
             public string Keyword;
             [Tooltip("The KeyCode to recognize.")]
             public KeyCode KeyCode;
-            [Tooltip("The UnityEvent to be invoked when the keyword is recognized.")]
-            public UnityEvent Response;
```

Comment:
This UnityEvent is basically the original point of the KeywordManager. It allows a developer to assign an action in the Inspector, instead of having to worry about the KeywordRecognizer's implementation in code.

There's another comment in here about leaving KeywordManager as is and creating a new class, which I support as well. If you're able to migrate this without losing functionality, then I'd support using KeywordManager.